doc/source/whatsnew/v1.1.4.rst

```diff
@@ -35,6 +35,7 @@ Bug fixes
 
 Other
 ~~~~~
+- Bug in :meth:`Series.isin` and :meth:`DataFrame.isin` raising a ``ValueError`` when the target was read-only (:issue:`37174`)
```

Comment:
why the other section?

Reply:
yes don't put in Other (delete this section); indexing would be fine